### ISSUE TITLE
Create a list of accessibility sims in chipper/data

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#611]
This will be helpful as accessibility starts to touch more and more sims. I'm thinking that this list will include all 'work in progress' sims rather than just finished/near finished ones.

**Comment by zepumph** [comment]
@jessegreenberg do you have any additions to this list (will be alphabatized)?

* JT
* BASE
* BAA
* OL
* RIAW
* Molarity
* Friction
* MAL
* Coulombs Law

Maybe the metric that we should be using is if we have started work on descriptions rather than if we have started work on keyboard nav, since keyboard nav enabled common code components are in all sims. In which case this would be the list:

* JT
* BASE
* BAA
* OL
* RIAW

What do you think?

**Comment by jessegreenberg** [comment]
Thanks @zepumph, I agree with your initial metric of "work in progress" sims.  Molarity, Friction, MAL, and Coulombs Law were also chosen for keyboard nav because they will get design time for description soon anyway.

**Comment by zepumph** [comment]
Done, closing
